hack/jenkins/job-configs/global.yaml

```diff
@@ -13,6 +13,17 @@
     publishers:
         - postbuildscript:
             builders:
+                # Dump any master/node logs that are still around at this point
+                - shell: |
+                    if [[ "${JOB_NAME}" =~ -e2e- && -d _artifacts ]]; then
+                        # This will be the case if we've checked out the repo directly
```

Comment:
I don't think this is necessary. On PR Jenkins we set `KUBE_RUN_FROM_OUTPUT=y` to force running from the tarballs, so we should always have a `kubernetes/cluster/log-dump.sh` script (assuming the branch is `master`).